addons/payment/wizards/payment_link_wizard.py

```diff
@@ -42,15 +42,33 @@ def default_get(self, fields):
     description = fields.Char('Payment Ref')
     access_token = fields.Char(compute='_compute_values')
     company_id = fields.Many2one('res.company', compute='_compute_company')
+    available_acquirer_ids = fields.Many2many(
+        comodel_name='payment.acquirer',
+        string="Payment acquirers available",
+        compute='_compute_payment_acquiers',
+        compute_sudo=True,
+    )
+    payment_acquirer_id = fields.Many2one(
```

Comment:
```suggestion
    acquirer_id = fields.Many2one(
```

Since the first part of model names refer to a module, we tend to use only the second part (I should add this to the guidelines :D)